src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/MainPageTests.java

```diff
@@ -1,22 +1,23 @@
-package com.wikia.webdriver.testcases.mercurytests;
+package com.wikia.webdriver.testcases.mercurytests.curatedcontenttests;
 
 import java.util.concurrent.TimeUnit;
 
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-import com.wikia.webdriver.common.contentpatterns.MercuryArticles;
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
+import com.wikia.webdriver.common.contentpatterns.MercurySubpages;
```

Comment:
Isn't two lines of `import com.wikia.webdriver.common.contentpatterns.MercurySubpages;` too much?